ui/src/app/applications/components/application-node-info/application-node-info.tsx

```diff
@@ -54,11 +54,77 @@ export const ApplicationNodeInfo = (props: {
     }
     if (props.live) {
         if (props.node.kind === 'Pod') {
-            const {reason, message} = getPodStateReason(props.live);
+            const {reason, message, netContainerStatuses} = getPodStateReason(props.live);
             attributes.push({title: 'STATE', value: reason});
             if (message) {
                 attributes.push({title: 'STATE DETAILS', value: message});
             }
+            if (netContainerStatuses.length > 0) {
+                attributes.push({
+                    title: 'CONTAINER STATE',
+                    value: (
+                        <div className='application-node-info__labels'>
+                            <table>
+                                <tbody>
```

Comment:
You can move out this code in a separate function. But could be done in follow-up PR

Reply:
will move